api/external/infra_proxy/request/orgs.proto

```diff
@@ -4,37 +4,53 @@ package chef.automate.api.infra_proxy.request;
 option go_package = "github.com/chef/automate/api/external/infra_proxy/request";
 
 message CreateOrg {
+    // Name of the Org.
     string name = 1;
+    // Admin user of the Org.
     string admin_user = 2;
+    // Admin Key of the Org.
```

Comment:
Please go through the capitalization. All things should be lowercase except the first word (like regular sentences).
```suggestion
    // Admin key of the organization.
```

Reply:
i think Organization should be capitalized here.. unless im missing something

Reply:
@susanev Using "Organization" would be inconsistent. We say, for example, "Name for the token" not "Name for the Token", and "Name for the project rule" not "Name for the project Rule", etc.

Reply:
okay, this is inconsistent with how we name things in the UI and the static docs, but if there's a different standard here then this is good with me